martin/src/cog/config.rs

```diff
@@ -0,0 +1,38 @@
+use std::{fmt::Debug, path::PathBuf};
+
+use serde::{Deserialize, Serialize};
+use url::Url;
+
+use crate::config::UnrecognizedValues;
+use crate::file_config::{ConfigExtras, SourceConfigExtras};
+use crate::{file_config::FileResult, Source};
+
+use super::source::CogSource;
+
+#[derive(Clone, Debug, Default, PartialEq, Serialize, Deserialize)]
+pub struct CogConfig {
```

Comment:
This is really weird (i understand it was a file move and we had this before, but still). Why would we have a configuration that does not actually contain any configuration?